packages/e2e/tests/_models/Installments.component.js

```diff
@@ -0,0 +1,24 @@
+import { t, Selector } from 'testcafe';
+
+export default class InstallmentsComponent {
+    constructor(baseEl = 'div.adyen-checkout__installments') {
+        this.baseEl = Selector(baseEl).with({ timeout: 0 });
```

Comment:
For some reason, if I don't set the `timeout: 0`, every time I invoke `selectInstallment()` or `selectRevolving()`  it takes `5s` for TestCafe to interact with the page elements.

Reply:
I have seen this kind of thing before - testcafe can be very slow to interact with things like radio buttons and checkboxes